app/mailers/prison_mailer.rb

```diff
@@ -107,4 +107,8 @@ def smoke_test?
   def visitors_email_address
     visit.visitors.first.email
   end
+
+  def formatted_date
+    l(Date.parse(@visit.slots.first.date), format: :short)
+  end
```

Comment:
I think it might be clearer to future readers if we used `I18n.l` here.

I also think we could rename the method to something like `formatted_first_slot_date` to explain what the date is.